src/arrayMethodJoin.js

```diff
@@ -4,8 +4,24 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let newString = '';
+
+    if (this.length === 1) {
```

Comment:
I am sure it is not correct statement https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
now understand, thanks